### DISCUSSION TITLE
Add API visibility for fees (offsite ads fee, transaction fee, listing auto-renewal fee)

### DISCUSSION THREAD

**Discussion by cervinakuy** [etsy/open-api#1089] [Ideas]
[disregard]

as per:
For those looking for detailed payment breakdowns, take a look at getShopPaymentAccountLedgerEntry & getShopPaymentAccountLedgerEntries on the API docs. I disregarded them at first, but they give you access to all shop payment data. Hopefully this helps someone out there!

**Comment by Dreamer2100** [comment]
Thank you so much, my friend
